pkg/datasource/wise/rates_request.go

```diff
@@ -0,0 +1,25 @@
+package wise
+
+import (
+	"github.com/c9s/requestgen"
+)
+
+// https://docs.wise.com/api-docs/api-reference/rate
+
+//go:generate requestgen -method GET -url "/v1/rates" -type RatesRequest -responseType []Rate
+type RatesRequest struct {
+	client requestgen.AuthenticatedAPIClient
+
+	source string  `param:"source"`
+	target string  `param:"target"`
+	time   *string `param:"time"`
```

Comment:
timeFormat attribute is supported, you can use that

Reply:
![image](https://github.com/c9s/bbgo/assets/50894/888a1317-db4e-4149-ada0-5bba8716329f)

Reply:
Wise use different time format `2006-01-02T15:04:05-0700` XD

Reply:
You can customize the format~

Reply:
ok

Reply:
You can set something like

timeFormat:"2006-01-02T15:04:05-0700"

Reply:
done